sql/telemetry_derived/deviations_v1/query.sql

```diff
@@ -6,4 +6,6 @@ SELECT
   metric
 FROM
   `moz-fx-data-shared-prod.analysis.deviations`
-WHERE date = @submission_date
+WHERE
+  metric IN ('desktop_dau', 'mean_active_hours_per_client')
```

Comment:
```suggestion
  -- We explicitly enumerate allowed metric types here so that we do not automatically
  -- publish new metric types publicly without review.
  metric IN ('desktop_dau', 'mean_active_hours_per_client')
```